src/main/resources/queries-mapping.json

```diff
@@ -1,31 +1,11 @@
 {
   "dynamic": false,
   "properties": {
-    "timestamp": {
-      "type": "date"
-    },
-    "query_id": {
-      "type": "keyword"
-    },
-    "query": {
-      "type": "text"
-    },
-    "query_response_id": {
-      "type": "keyword"
-    },
-    "query_response_object_ids": {
-      "type": "keyword"
-    },
-    "user_id": {
-      "type": "keyword"
-    },
-    "user_query": {
-      "type": "text",
-      "fields": {
-        "keyword": {
-          "type": "keyword"
-        }
-      }
-    }
+    "timestamp": { "type": "date" },
+    "query_id": { "type": "keyword", "ignore_above": 100 },
+    "query": { "type": "flat_object" },
+    "query_response_id": { "type": "keyword", "ignore_above": 100 },
```

Comment:
we should probably get on the same page on what query_response_id is, again, we odn't have it in the spec, is this an internal thing?

Reply:
This is a difference in the mappings vs the request/response schema. This is for indexing the `query_response_id`, not returning it in a response.

Reply:
I don't like it, but @jzonthemtn thought it was an explicit requirement.

Reply:
`query_response_id` is a unique id per OS response to the client.
Theoretically, a client could reuse the same `query_id` several times to query something, but the `query_response_id` should always be unique.